pkg/nodeconfig/nodeconfig.go

```diff
@@ -467,15 +467,18 @@ func (nc *nodeConfig) Deconfigure() error {
 		return errors.Wrap(err, "error deconfiguring instance")
 	}
 
-	// Clear the version annotation from the node object to indicate the node is not configured
-	patchData, err := metadata.GenerateRemovePatch([]string{}, []string{metadata.VersionAnnotation})
-	if err != nil {
-		return errors.Wrapf(err, "error creating version annotation remove request")
-	}
-	_, err = nc.k8sclientset.CoreV1().Nodes().Patch(context.TODO(), nc.node.GetName(), kubeTypes.JSONPatchType,
-		patchData, meta.PatchOptions{})
-	if err != nil {
-		return errors.Wrapf(err, "error removing version annotation from node %s", nc.node.GetName())
+	// Check version annotation and remove iff present
```

Comment:
How does this commit relate to adding Server 2022 support?

Reply:
Removed.